### PR TITLE
Submit actions

### PR DESCRIPTION
**Pull request by danielnaab**
This branch continues work on #222, adding the ability to register multiple form submission handlers on the backend. Additionally, a "package download" pattern was added, which consists of a "Download PDF" submit button.

Actual PDF downloading is not working, due to binary/JSON serialization issues. There will be a follow-up PR to store PDF binaries in the database, outside of the form blueprint object.

This also unblocks #310.

### CODE REVIEW

File: packages/forms/src/patterns/input/builder.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { PatternBuilder } from '../../pattern';
+import { type InputPattern } from '.';
+
+export class Input extends PatternBuilder<InputPattern> {
```
Comment (ethangardner) [inline packages/forms/src/patterns/input/builder.ts:4]: I love the concept of the builder class here. Could we document the reasoning in an ADR?

File: packages/forms/src/submission.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    const handler = this.handlers[result.data.handlerId];
+    if (handler === undefined) {
+      return failure(
+        `Submission handler with id ${result.data.handlerId} does not exist`
+      );
+    }
+    const pattern = getPattern(form, result.data.patternId);
+    if (pattern === undefined) {
+      return failure(
+        `asdfPattern with id ${result.data.patternId} does not exist`
```
Comment (ethangardner) [inline packages/forms/src/submission.ts:79]: Home row for the win 🥇 :)

File: packages/forms/src/patterns/input/builder.ts (hunk 1 above)
Comment (danielnaab) [reply packages/forms/src/patterns/input/builder.ts:4]: Will do. As it's more technical, I think it might read better as README text.

**Review by ethangardner — APPROVED** [review]
This is good to go! Thanks for these changes.

**Pull request merged**